feedbackMessage.js

```diff
@@ -0,0 +1,83 @@
+const buildFeedbackMessage = (stylelintOutcomes, root) => {
+  const { issues, errorCount, warningCount } = extractStylelintIssues(stylelintOutcomes);
+  const { errorIssues, warningIssues } = splitIssues(issues, root);
+
+  let feedbackMessage = [];
+
+  feedbackMessage.push(feedBackTitleMessage('erro', errorCount));
+  feedbackMessage = addIssueLineMessage(feedbackMessage, errorIssues, true);
+
+  feedbackMessage.push(feedBackTitleMessage('aviso', warningCount));
+  feedbackMessage = addIssueLineMessage(feedbackMessage, warningIssues);
+
+  return feedbackMessage.join('\n');
+};
+
+const extractStylelintIssues = (stylelintOutcomes) => {
+  return stylelintOutcomes.reduce((total, currentValue) => {
+    const { source, warnings } = currentValue;
+    let issues = {
+      source,
+      error: [],
+      warning: [],
+    };
+
+    warnings.forEach(({ line, column, severity, text }) => (
+      issues[severity].push({ line, column, text })
+    ));
+
+    return {
+      issues: total.issues.concat(issues),
+      errorCount: total.errorCount + issues.error.length,
+      warningCount: total.warningCount + issues.warning.length,
+    };
+  }, { issues: [], errorCount: 0, warningCount: 0 });
+};
+
+const splitIssues = (issues, root) => {
+  return issues.reduce((total, currentValue) => {
+    let errorIssues = [];
+    let warningIssues = [];
+    const { source, error, warning } = currentValue;
+
+    errorIssues = buildIssuesMessage(source, error, root);
+    warningIssues = buildIssuesMessage(source, warning, root);
```

Comment:
Você poderia reduzir essas linhas para:
```suggestion
    const { source, error, warning } = currentValue;
    const errorIssues = buildIssuesMessage(source, error, root);
    const warningIssues = buildIssuesMessage(source, warning, root);
```